pkg/cmd/start/start.go

```diff
@@ -183,6 +186,15 @@ func (o *Options) Validate() error {
 		o.customParameterMap[paths[0]] = paths[1]
 	}
 
+	o.customEnvMap = map[string]string{}
+	for _, customEnv := range o.CustomEnvs {
+		envs := strings.SplitN(customEnv, "=", 2)
+		if len(envs) != 2 {
+			return options.InvalidOptionf("env", customEnv, "should be of the form 'name=value'")
+		}
+		o.customEnvMap[envs[0]] = envs[1]
+	}
+
```

Comment:
Code is not needed. You can read the option value to a map automatically by using `StringToStringVarP` instead of `StringArrayVarP`

Reply:
Thanks!
Resolved via https://github.com/jenkins-x-plugins/jx-pipeline/pull/510/commits/9b202adc9a874f6bf4927f2423149975643053ec

Then maybe `StringToStringVarP` can also be used for parameters here
https://github.com/jenkins-x-plugins/jx-pipeline/blob/main/pkg/cmd/start/start.go#L150

Reply:
It could, yeah. If you like to change that you can.